cmd/secrets-provider/main.go

```diff
@@ -120,29 +120,26 @@ func processAnnotations(ctx context.Context, tracer trace.Tracer) error {
 
 func secretRetriever(ctx context.Context,
 	tracer trace.Tracer) (*conjur.SecretRetriever, error) {
-
-	// Create a trace Span
-	spanCtx, span := tracer.Start(ctx, "Create Conjur secret retriever")
+	// Gather K8s authenticator config
```

Comment:
Is the intention there authn-k8s or configuration gathering is general for both authn's?

Reply:
General i will fix the comment